static/app/views/issueDetails/streamline/sidebar/firstLastSeenSection.spec.tsx

```diff
@@ -0,0 +1,160 @@
+import {GroupFixture} from 'sentry-fixture/group';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {ProjectFixture} from 'sentry-fixture/project';
+import {ReleaseFixture} from 'sentry-fixture/release';
+
+import {render, screen, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import {useFetchAllEnvsGroupData} from 'sentry/views/issueDetails/groupSidebar';
+import FirstLastSeenSection from 'sentry/views/issueDetails/streamline/sidebar/firstLastSeenSection';
+import {useEnvironmentsFromUrl} from 'sentry/views/issueDetails/utils';
+
+jest.mock('sentry/views/issueDetails/utils', () => ({
+  useEnvironmentsFromUrl: jest.fn(),
+}));
+jest.mock('sentry/views/issueDetails/groupSidebar', () => ({
+  useFetchAllEnvsGroupData: jest.fn(),
+}));
+
+const mockUseEnvironmentsFromUrl = useEnvironmentsFromUrl as jest.MockedFunction<
+  typeof useEnvironmentsFromUrl
+>;
+const mockUseFetchAllEnvsGroupData = useFetchAllEnvsGroupData as jest.MockedFunction<
+  typeof useFetchAllEnvsGroupData
+>;
+
+describe('FirstLastSeenSection', () => {
+  const organization = OrganizationFixture();
+  const project = ProjectFixture();
+  const group = GroupFixture({
+    project,
+    firstSeen: '2024-01-06T10:00:00Z',
+    lastSeen: '2024-01-14T12:00:00Z',
+  });
+
+  const firstRelease = ReleaseFixture({version: '1.0.0'});
+  const lastRelease = ReleaseFixture({version: '1.1.0'});
+
+  let mockFirstLastRelease: jest.Mock;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    mockUseEnvironmentsFromUrl.mockReturnValue([]);
+    mockUseFetchAllEnvsGroupData.mockReturnValue({
+      data: GroupFixture({
+        firstSeen: '2024-01-06T10:00:00Z',
+        lastSeen: '2024-01-14T12:00:00Z',
+      }),
+      isError: false,
+      error: null,
+      isPending: false,
+      isLoading: false,
+      isSuccess: true,
+      isFetching: false,
+      isRefetching: false,
+      isStale: false,
+      refetch: jest.fn(),
+    } as any);
+
+    MockApiClient.clearMockResponses();
+
+    mockFirstLastRelease = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/issues/${group.id}/first-last-release/`,
+      method: 'GET',
+      body: {
+        id: group.id,
+        firstRelease,
+        lastRelease,
+      },
+    });
+  });
+
+  it('renders first and last seen information with release data', async () => {
+    render(<FirstLastSeenSection group={group} />, {organization});
+
+    await waitFor(() => {
+      expect(mockFirstLastRelease).toHaveBeenCalled();
+    });
+
+    expect(mockFirstLastRelease).toHaveBeenCalledWith(
+      `/organizations/${organization.slug}/issues/${group.id}/first-last-release/`,
+      expect.objectContaining({
+        query: {},
+      })
+    );
+
+    expect(await screen.findByText('First seen')).toBeInTheDocument();
+    expect(screen.getByText('Last seen')).toBeInTheDocument();
+
+    expect(
+      await screen.findByText((_content, element) => {
```

Comment:
we should probably use `textWithMarkupMatcher` or something similar rather than trying to break it down by `_content` and `element` (not a pattern i remember seeing in our frontend tests)